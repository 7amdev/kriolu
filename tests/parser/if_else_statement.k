si (verdadi) {
    imprimi "verdadi!";
} sinou {
    imprimi "falsu!";
}