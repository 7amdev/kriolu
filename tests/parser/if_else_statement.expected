<statement si>
<condition>
    verdadi
<then>
  <block>
    <imprimi>
      'verdadi!'
<else>
  <block>
    <imprimi>
      'falsu!'